src/bin/stratisd.rs

```diff
@@ -75,15 +69,21 @@ fn main() {
         .get_matches();
 
     if matches.is_present("debug") {
-        unsafe { DEBUG = true }
-    };
+        let mut builder = LogBuilder::new();
```

Comment:
error! is always on... from the docs:

Log levels are controlled on a per-module basis, and by default all logging is disabled except for error!.